server/rucio.conf.j2

```diff
@@ -20,6 +20,10 @@ LogFormat "%h\t%t\t%{X-Rucio-Forwarded-For}i\t%T\t%D\t\"%{X-Rucio-Auth-Token}i\"
 LoadModule authn_core_module modules/mod_authn_core.so
 LoadModule cache_disk_module modules/mod_cache_disk.so
 
+WSGIRestrictEmbedded On
+WSGIDaemonProcess rucio processes=4 threads=4
```

Comment:
I wonder if these numbers should be made configurable. 4/4 is the config that works well for our standard CERN OpenStack VMs.

Reply:
sure, I can add that